src/test/java/de/tum/in/www1/artemis/assessment/ParticipantScoreIntegrationTest.java

```diff
@@ -162,17 +135,12 @@ void setupTestScenario() {
         participationUtilService.createParticipationSubmissionAndResult(idOfTeamTextExercise, team, 10.0, 10.0, 50, true);
 
         // setting up exam
-        Exam exam = ExamFactory.generateExam(course);
+        exam = examUtilService.addExamWithUser(course, student1, true, pastTimestamp, pastTimestamp, pastTimestamp);
         ExamFactory.generateExerciseGroup(true, exam);
```

Comment:
Do we still need this line since you should generate the exercise group with the statement above?

Reply:
No, we don't. I removed it now and the corresponding tests still pass :+1: